src/scripts/main.js

```diff
@@ -21,7 +21,28 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const dataKeys = Object.keys(data);
+
+  for (const key of dataKeys) {
+    const li = document.createElement('li');
+    const ul = document.createElement('ul');
+
+    li.innerHTML = key;
+
+    if (Object.entries(data[key]).length !== 0) {
```

Comment:
```suggestion
    if (Object.entries(data[key]).length !== 0) {
          const ul = document.createElement('ul');
```

Reply:
why is this necessary?

Reply:
To avoid creating unneeded ul and create an ul only if `Object.entries(data[key]).length !== 0`

Reply:
ok, thanks for the explonation